index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Israel CHIZUNGU Portfolio</title>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <section>
+        <!--  toolbar starts -->
+        <header>
+            <a href="#" class="logo">My Logo</a>
+            <a href="#"><img src="img/hamburger.png" alt="humburger icon" class="hamburger"></a>
+            
+
+        </header>
+        <!--  toolbar ends -->
+
+
+        <!--  headline starts -->
+        <main>
+            <h2 class="intro-text">I’m Mariane
+                Glad to see you!</h2>
+            <p class="description">I’m a software developer! I can help you build a product , feature or website Look through some of my
+                work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.</p>
+            <a href="#" class="subtitle">LET’S CONNECT</a>
+            <div class="links-group">
+                <a href="#" class="link"><img src="img/Vectortwitter.svg" alt="twitter" class="img-link"></a>
+                <a href="#" class="link"><img src="img/Linkedin iconlinkedin.svg" alt="Linkedin" class="img-link"></a>
+                <a href="#" class="link"><img src="img/Vectorm.svg" alt="M" class="img-link"></a>
+                <a href="#" class="link"><img src="img/Vectorgithub.svg" alt="github" class="img-link"></a>
+                <a href="#" class="link"><img src="img/Vectorp-l.svg" alt="Peace & Love" class="img-link"></a>
+            </div>
+        </main>
```

Comment:
- [ ] As required for the project and for semantics reasons, kindly make sure the **headline section** has a `<section>` tag as a container and not the `<main>` tag(it represents a generic standalone section of the document). The `<main>` tag can wrap all the sections of your project where `<section>` tags are sub-tags of the `<main>` tag.
Here is an example of the good syntax:
```HTML
  <header>
    code of the header here...
  </header>

  <main>
    <section>
      code for the 1st section...
    </section>

    <section>
      code for the 2nd section...
    </section>
    ...
  </main>

  <footer>
    code for the footer
  </footer>
```